plugin.video.mlbtv/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2016.7.28" provider-name="eracknaphobia">
+  <requires>
+   <import addon="xbmc.python" version="2.19.0"/>   
```

Comment:
Apparently I should just just jump it to 2.25 for krypton. I will address the issues you mentioned below and resubmit to the krypton branch once completed. @enen92